forc-gm/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "forc-gm"
+version = "0.1.0"
```

Comment:
If this is going to be published (not sure if we want it to?) then it needs to have the same `version` as other published crates.

Reply:
Does seem like an unnecessary crate to publish, since it would just be a template...

Only upside would be that it allows people to do `cargo install forc-gm` easily to test it out, however on second thought, maybe it should remain unpublished since there's so few lines anyway. I'll update it with `publish = false` for now and bump the version down.

Reply:
We should potentially just publish this even once, if only to stop any malicious actors from potentially using the name in the future.